### PR TITLE
Rails 8

### PR DESCRIPTION
**Pull request by chao-xian**
### What problem does this pull request solve?

Deployed to Dev and the [e2e tests pass](https://eu-west-2.console.aws.amazon.com/codesuite/codebuild/498160065950/projects/forms-api-e2e-tests-dev/build/forms-api-e2e-tests-dev%3A7d5cc78a-fd7b-4154-aadb-988b4abb9edb?region=eu-west-2)

Trello card: https://trello.com/c/l1aS3Mq6/2598-rails-7-to-8-upgrade-for-forms-api

### Things to consider when reviewing

- Ensure that you consider the wider context.
- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -17,44 +15,41 @@
...
-    config.public_file_server.headers = { "Cache-Control" => "public, max-age=#{2.days.to_i}" }
+    config.public_file_server.headers = { "cache-control" => "public, max-age=#{2.days.to_i}" }
   else
     config.action_controller.perform_caching = false
-
-    config.cache_store = :null_store
   end
 
+  # Change to :null_store to avoid any caching.
+  config.cache_store = :memory_store
```
Comment (DavidBiddle) [inline config/environments/development.rb:27]: Is this change intentional? I'm not sure how big a difference it makes but I'm wondering if we set it to `:null_store` for a reason

Comment (chao-xian) [reply config/environments/development.rb:27]: yeh https://railsdiff.org/7.2.2/8.0.0#diff-4cc21c4ebed19383e79b7e232d2be4b0e6b154f2

**Review by DavidBiddle — APPROVED** [review]
Changes look good and no issues locally 🎉
